velox/functions/lib/aggregates/BitwiseAggregateBase.h

```diff
@@ -72,6 +72,7 @@ class BitwiseAggregateBase : public SimpleNumericAggregate<T, T, T> {
 template <template <typename U> class T>
 exec::AggregateRegistrationResult registerBitwise(
     const std::string& name,
+    bool ignoreDuplicates,
```

Comment:
Isn't this flag always true? Do we need this parameter?

Reply:
It's not true for xor